### PR TITLE
add the manifest

### PR DESCRIPTION
**Pull request by wenchajun**
Signed-off-by: chengdehao <dehaocheng@kubesphere.io>

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (zhu733756) [inline .idea/.gitignore]: .idea/** is needed ?

**Review by zhu733756 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -885,7 +885,7 @@ metadata:
   name: fluentd-only-stdout
   namespace: fluent
   labels:
-    output.fluentd.fluent.io/enabled: "true"
+    output.fluentd.fluent.io/mode: "true"
```
Comment (benjaminhuo) [inline README.md]: `output.fluentd.fluent.io/mode: "true"` should be changed to `filter.fluentd.fluent.io/mode: "fluentd-only"
` for both the selector and output labels

File: manifest/fluentbit-fluend/cluster-namespace-mix/fluentbit-config.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
```
Comment (benjaminhuo) [inline manifest/fluentbit-fluend/cluster-namespace-mix/fluentbit-config.yaml]:
```shell
fluentbit-only-mode
- kubelet-log-to-es
- k8s-app-log-to-kafka
fluentbit-to-fluentd-mode
- to-cluster-wide-fluentd
- to-namespaced-wide-fluentd	
- to-cluster-namespaced-mixed-fluentd
- to-multi-tenant-fluentd
- output-to-different-kafka-topics-with-fluentd	
- use-buffer-in-fluentd-output
fluentd-only-mode
- receive-logs-from-http
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: manifest/fluentbit-to-fluentd-mode/output-to-different-kafka-topics-with-fluentd/fluentbit-config.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterFluentBitConfig
```
Comment (benjaminhuo) [inline manifest/fluentbit-to-fluentd-mode/output-to-different-kafka-topics-with-fluentd/fluentbit-config.yaml]: We needn't add the same fluentbit stuff into every subfolder of `fluentbit-to-fluentd-mode` repeatedly.
And the same is for the file fluentbit-forward.yaml

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**